httplug/exceptions.rst

```diff
@@ -0,0 +1,20 @@
+Exceptions
+==========
+
+HTTPlug defines one Exception interface: ``Http\Client\Exception``. All HTTP
+client exceptions must implement this interface.
+
+``HttpClient::sendRequest()`` can throw one of the following exceptions.
+
+================================== ====================== ===================
+Exception                          Thrown when            Methods available
+================================== ====================== ===================
+TransferException                  ???
+└ RequestException                 the request is invalid ``getRequest()``
+ |nbsp| |nbsp| └ NetworkException  no response received
+                                   due to network issues  ``getRequest()``
+ |nbsp| |nbsp| └ HttpException     error response         ``getRequest()``
+                                                          ``getResponse()``
+================================== ====================== ===================
```

Comment:
I think we already went down this path. TransferException is used somewhere, don't ask where, I don't know. :stuck_out_tongue: